sips/sip-submission.md

```diff
@@ -1,61 +1,275 @@
+The aim of the Scala Improvement Process is to apply the openness and
+collaboration that have shaped Scala's documentation and implementation to the
+process of evolving the language. This document captures our guidelines,
+commitments and expectations regarding this process.
 
-## How do I submit? ##
+## Why write a SIP?
 
-The process to submit is simple:
+SIPs are key to making Scala better for the good of everyone. If you decide to
+invest the time and effort of putting a SIP forward and seeing it through, your
+efforts and time will shape and improve the language, which means that your
+proposal may impact the life of a myriad of developers all over the world,
+including those on your own team. For many, this aspect alone can be quite
+worthwhile.
 
-* Fork the Scala documentation repository, [http://github.com/scala/scala.github.com](http://github.com/scala/scala.github.com).
-* Create a new SIP file in the `sips/pending/_posts/`.  Check the [Writing a SIP Tutorial](sip-tutorial.html)
-  * Make sure the new file follows the format:  `YYYY-MM-dd-{title}.md`.  Use the proposal date for `YYYY-MM-dd`.
-  * Use the [Markdown Syntax](http://daringfireball.net/projects/markdown/syntax) to write your SIP.
-  * Follow the instructions in the [README](https://github.com/scala/scala.github.com/blob/gh-pages/README.md) to build your SIP locally so you can ensure that it looks correct on the website.
-* Create a link to your SIP in the "pending sips" section of `index.md`
-* Commit your changes to your forked repository
-* Create a new [pull request](https://github.com/scala/scala.github.com/pull/new/gh-pages).  This will notify the Scala SIP team.
+However, it's important to note that seeing a SIP through to its conclusion is
+an involved task. On the one hand, it takes time to convince people that your
+suggestions are a worthwhile change for hundreds of thousands of developers to
+accept. Particularly given the sheer volume of developers that could be affected
+by your SIP, SIP acceptance is conservative and carefully thought through.
+Typically, this includes many rounds of discussion with core Scala maintainers
+and the overall community, several iterations on the design of the SIP, and some
+effort at prototyping the proposed change. Often, it takes months of discussion,
+re-design, and prototyping for a SIP to be accepted and included in the Scala
+compiler. It is, therefore important to note that seeing a SIP through to its
+conclusion can be time-consuming and not all SIPs may end up in the Scala
+compiler, although they may teach us all something!
+
+If you’re motivated enough to go through this involved but rewarding process, go
+on with writing and keep on reading.
+
+## What's the process for submitting a SIP?
+
+There are four major steps in the SIP process:
+
+1. Initial informal discussion (2 weeks)
+2. Submission
```

Comment:
Sounds good, but it will give the impression that the author has to meet a deadline (say, 1 month) when he doesn't, it's the Process Lead who does. If we add a time reference, then we should reword the title to 'Submission and Validation'.